kclvm/runner/src/lib.rs

```diff
@@ -1,2 +1,42 @@
+use kclvm_ast::ast::Program;
+use kclvm_sema::resolver::scope::ProgramScope;
+use runner::{ExecProgramArgs, KclvmAssembler, KclvmLinker, KclvmRunner, KclvmRunnerOptions};
+
 pub mod command;
 pub mod runner;
+
+#[cfg(test)]
+mod tests;
+
+/// Returns the KCL program executing result as Result< a_json_string, an_err_string >.
+/// program: ast.Program returned by kclvm-parser.
+/// scope: ProgramScope returned by kclvm-sema
+///
+/// In the method, dylibs is generated by KclvmAssembler, and method "KclvmAssembler::gen_dylibs"
+/// will return dylibs path in a "Vec<String>";
+///
+/// After linking all dylibs by KclvmLinker, method "KclvmLinker::link_all_dylibs" will return a path
+/// for dylib.
+///
+/// At last, KclvmRunner will be constructed and call method "run" to execute the kcl program.
+pub fn execute(
```

Comment:
1. Add some Samples directly in the doc comment for testing
2. This end-to-end function can add some benchmarks